haproxy/dockerdir/etc/haproxy-auto.cfg

```
@@ -0,0 +1,20 @@
+    global
+      maxconn 100
```

proxysql default 2048
```suggestion
      maxconn 2048
```

---

done